gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -355,7 +355,7 @@
 
   /** Configuration key for the connect timeout (in millisecond) for gRPC read requests to GCS. */
```

Comment:
we should update here and in the documentation that the actual timeout will factor the object size. And this value is adding a buffer over that.

Reply:
Done